samples/sqlservices/.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 *.vsix
 typings/*
+out
```

Comment:
slide this in, so that I don't get tons of changes in github when I build this extension